hotel contract.js

```diff
@@ -24,11 +24,11 @@ contract = class {
 }
 
 // Yeni otelleri ve propertylerini tanımlıyorum. N tane olabilir ama örnek için sadece 2 tane tanımlıyorum.
```

Comment:
Try to use only English in your comments and variable names

Reply:
Ok I will use english comments

Reply:
Ok then. I approve the pr but please change comments in the next pr.